composer.json

```diff
@@ -146,26 +146,25 @@
             "drupal/bootstrap_barrio": {
                 "Js error if table not found (3519060)": "https://www.drupal.org/files/issues/2025-04-14/bootstrap-barrio-table-not-found-3519060-1.patch"
             },
-            "drupal/cas": {
-                "ServiceNotFoundException when logging out (2948185)": "https://www.drupal.org/files/issues/cas_logout_error-2948185-2.patch"
-            },
             "drupal/config_distro": {
                 "Storage comparer issue": "https://gist.githubusercontent.com/tadean/e2e0dd57f1a9c4eae28d8e713cb01d82/raw/e114987d4a76c183208c763de4dbfecbf41b7f35/config_distro_storage_comparer_issue.patch"
             },
+            "drupal/config_provider": {
+                "Drupal 11.2 compatibility issues (3532693)": "https://git.drupalcode.org/project/config_provider/-/merge_requests/20.diff"
```

Comment:
@joeparsons do you think the maintainers :stuck_out_tongue: should determine which path to take on this Drupal.org MR before merging this and make a release instead of applying a patch?

I don't think it's necessarily a blocker - if not we'd probably just want to use a gist.